Python/tests/conftest.py

```diff
@@ -6,13 +6,13 @@
 from sqlmlutils import ConnectionInfo, Scope
 
 driver = os.environ['DRIVER'] if 'DRIVER' in os.environ else "SQL Server"
-server = os.environ['SERVER'] if 'SERVER' in os.environ else "localhost"
+server = os.environ['SERVER'] if 'SERVER' in os.environ else "sqlmlutils-mi-server.public.0c3d2187285f.database.windows.net,3342"
 database = os.environ['DATABASE'] if 'DATABASE' in os.environ else "AirlineTestDB"
-uid = os.environ['USER'] if 'USER' in os.environ else ""
-pwd = os.environ['PASSWORD'] if 'PASSWORD' in os.environ else ""
+uid = os.environ['USER'] if 'USER' in os.environ else "extensibilityuser"
+pwd = os.environ['PASSWORD'] if 'PASSWORD' in os.environ else "cereal-whizzing-salute-900!@#"
 
 uidAirlineUser = "AirlineUserdbowner"
-pwdAirlineUser = os.environ['PASSWORD_AIRLINE_USER'] if 'PASSWORD_AIRLINE_USER' in os.environ else "FakeT3sterPwd!"
+pwdAirlineUser = os.environ['PASSWORD_AIRLINE_USER'] if 'PASSWORD_AIRLINE_USER' in os.environ else "testing!123"
```

Comment:
keeping a password value in the file is not recommended, can we use azure key vault ?